### PR TITLE
add pump sensors

### PR DESCRIPTION
**Pull request by jr1221**
## Changes

_Explanation of changes goes here_

## Notes

_Any other notes go here_

## Test Cases

- Case A
- Edge case
- ...

## To Do

_Any remaining things that need to get done_

- [ ] item 1
- [ ] ...

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please reach out to your Project Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [ ] No merge conflicts
- [ ] All checks passing
- [ ] Remove any non-applicable sections of this template
- [ ] Assign the PR to yourself
- [ ] Request reviewers & ping on Slack
- [ ] PR is linked to the ticket (fill in the closes line below)

Closes # (issue #)

### CODE REVIEW

**Comment by jr1221** [comment]
Cloes #262

File: cangen/can-messages/mpu.json [hunk 1]
```diff
@@ -1434,7 +1434,7 @@
     "desc": "Pump Sensors Voltage",
     "points": [
       {
-        "size": 32,
+        "size": 16,
         "endianness": "little",
         "format": "divide10000",
```
Comment (bjackson312006) [inline cangen/can-messages/mpu.json:1439]: This should be divide1000

File: cangen/can-messages/mpu.json [hunk 2]
```diff
@@ -1445,7 +1445,7 @@
         }
       },
       {
-        "size": 32,
+        "size": 16,
         "endianness": "little",
         "format": "divide10000",
```
Comment (bjackson312006) [inline cangen/can-messages/mpu.json:1450]: Also should be divide1000

**Review by bjackson312006 — CHANGES_REQUESTED** [review]
